src/main/java/uk/gov/register/core/Item.java

```diff
@@ -20,6 +20,7 @@
     private final JsonNode content;
     private final HashValue v1HashValue;
     private final HashValue blobHash;
+    private final Optional<Integer> itemOrder;
```

Comment:
it does seem a bit weird that this is a property of the Item itself rather than the view where it is paginated. But happy to go with this for now and re-visit if we get time.